src/momento/config/configurations.py

```diff
@@ -35,7 +35,14 @@ def v1() -> Configurations.Laptop:
             This configuration is guaranteed not to change in future releases of the Momento Python SDK.
             """
             return Configurations.Laptop(
-                StaticTransportStrategy(StaticGrpcConfiguration(timedelta(seconds=15))),
+                StaticTransportStrategy(
+                    StaticGrpcConfiguration(
+                        deadline=timedelta(seconds=15),
+                        keepalive_permit_without_calls=True,
+                        keepalive_time=timedelta(milliseconds=5000),
+                        keepalive_timeout=timedelta(milliseconds=1000),
```

Comment:
It looks like these three settings are identical to every instance except one. I recommend making these the default values in the constructor then and we can specialize in just the instance we make for the lambda config.